.github/workflows/update-lts-tags.yaml

```diff
@@ -35,6 +40,11 @@ jobs:
           cd src/test/scripts
           python update_versions.py
 
+      - name: Update README files with helm-docs
+        run: |
+          cd src/main/charts
+          helm-docs
```

Comment:
I think we need to add the template file argument here. We define it for the pre-commit hook as well: https://github.com/atlassian/data-center-helm-charts/blob/main/.pre-commit-config.yaml#LL11C24-L11C24

```
          - --template-files=./README.md.gotmpl
```

Reply:
@nanux yes, but we use the default tpl name, and:

```
  -t, --template-files strings       gotemplate file paths relative to each chart directory from which documentation will be generated (default [README.md.gotmpl])
```

Reply:
So we can remove it from the `pre-commit-config.yaml`?

Reply:
As long as our tpl file matches the default that helm-docs cli has